contrib/bootstrap/README.md

```diff
@@ -0,0 +1,25 @@
+Generating Bootstrap
+====================
+
+We use bootstrap's customizer to ensure our bootstrap is the small and also
+stays with the aesthetics of debian.
+
+The file `config.json` contains related configuration.
+
+How To Generate
+-----------------
+
+- Go to https://getbootstrap.com/customize/
+
+- Upload the new `config.json` file, making sure you change the one in
+  `/contrib/bootstrap` to match.
+
+- Download the `bootstrap.zip` file
+  ```sh
+  BASE=/path/to/debsouces/repo # make sure this points to top level
+  mkdir tmp
+  unzip ~/Downloads/bootstrap.zip -d ./tmp # Download location may vary
```

Comment:
Good point. Will fix.
On 10 Jun 2016 22:38, "Orestis Ioannou" <notifications@github.com> wrote:

> In contrib/bootstrap/README.md
> <https://github.com/Debian/debsources/pull/61#discussion_r66684675>:
>
> > +
> > +The file `config.json` contains related configuration.
> > +
> > +How To Generate
> > +-----------------
> > +
> > +- Go to https://getbootstrap.com/customize/
> > +
> > +- Upload the new `config.json` file, making sure you change the one in
> > +  `/contrib/bootstrap` to match.
> > +
> > +- Download the `bootstrap.zip` file
> > +  ```sh
> > +  BASE=/path/to/debsouces/repo # make sure this points to top level
> > +  mkdir tmp
> > +  unzip ~/Downloads/bootstrap.zip -d ./tmp # Download location may vary
>
> why not in /tmp ? this would simplify things (avoid creating a dir and
> then needing to remove it ...)
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/Debian/debsources/pull/61/files/b64d878f0416a7d9f967428bd8c38f18620c3223#r66684675>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AB1bp7B5HJRDmJSaooY_UV2lYtLoBV_jks5qKdlIgaJpZM4Inw_q>
> .
>